relay-server/src/actors/envelopes.rs

```diff
@@ -997,7 +991,11 @@ impl EnvelopeProcessor {
         } else if let Some(mut item) = raw_security_item {
             relay_log::trace!("processing security report");
             state.sample_rates = item.take_sample_rates();
-            self.event_from_security_report(item)?
+            let result = self.event_from_security_report(item);
+            if let Err(ref error) = result {
+                relay_log::error!("failed to extract security report: {}", LogError(error));
+            }
+            result?
```

Comment:
I would have thought you would use
```
self.event_from_security_report(item).map_err(|err| { 
  relay_log::error!("failed to extract security report: {}", LogError(error));
  err
})
```

Reply:
Do you prefer this? I thought about it, since we had done this before. It's also one line shorter. However, we're not really mapping an error here, so it seemed a bit indirect.

Reply:
I don't have a strong preference, but I think we generally do map_err, I'm fine with both

Reply:
Let me change to map_err then.